### ISSUE TITLE
[feature] Support for GitHub Attestation Store

### ISSUE THREAD

**Issue by ianlewis** [slsa-framework/slsa-github-generator#3669]
We should consider adding support for storing results in GitHub's attestation store.

I think, we would need to create a predicate that matches what the attestation store expects, so it would be slightly different than the ones we create today. This might be ok, since we aren't supporting SLSA v1.0 much in our workflows yet, so we could do it at the same time.

**Comment by ianlewis** [comment]
@OpenWaygate So I guess you generating two attestations? one created by `slsa-github-generator` and one by the `attest-build-provenance` action?

**Comment by OpenWaygate** [comment]
yes, while the one created by `slsa-github-generator` will appear in release assets, the one created by `attest-build-provenance` will appear in https://github.com/username/projectname/attestations
